### PR TITLE
RSpec v3.1.0

### LINKED ISSUE #3: CommandLine Error
/Library/Ruby/Gems/2.0.0/gems/parallel_split_test-0.3.0/lib/parallel_split_test/command_line.rb:8:in `<module:ParallelSplitTest>': uninitialized constant RSpec::Core::CommandLine (NameError)

### PR DESCRIPTION
**Pull request by bootstraponline**
Fix #3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 spec/tmp*
+.idea/
+pkg/
```
Comment (grosser) [inline .gitignore]: if you put this into your global .gitignore you will never have to add it to any project https://github.com/grosser/dotfiles/blob/master/gitconfig#L66-L67

**Comment by grosser** [comment]
yeah! :D

**Comment by grosser** [comment]
0.4.0 is all yours :)

**Comment by bootstraponline** [comment]
awesome!

File: .gitignore (hunk 1 above)
Comment (bootstraponline) [inline .gitignore]: cool, I'll setup a global .gitignore. thanks for the tip.

**Pull request merged**